README.md

```diff
@@ -95,8 +95,6 @@ ayesql is another Elixir library, a bit more powerful than yesql:
 - Special constructs for [query composability](https://hexdocs.pm/ayesql/readme.html#query-composition) in SQL files.
 - Special constructs for [optional parameters](https://hexdocs.pm/ayesql/readme.html#optional-parameters) in SQL files.
 
-Should I use ayesql then? Not necessarily... it may depend on your style preference.
-
```

Comment:
Removed this sentence, but feel free to point out if anything feels off with the wording or examples.